script.artistslideshow/addon.xml

```diff
@@ -10,6 +10,8 @@
 		<provides>executable</provides>
 	</extension>
 	<extension point="xbmc.addon.metadata">
+		<news>2.0.3
+		- removed support for htbackdrops.org (API access no longer avaialble)</news>
```

Comment:
That's the new US spelling of available.  '-)  Extra letters make it more better.

Do you want me to resubmit?